chrome/content/zotero/zoteroPane.js

```diff
@@ -1256,6 +1256,28 @@ var ZoteroPane = new function()
 						// Keep whatever else is there
 						+ ' ' + menupopup.getAttribute('onpopuphidden');
 				}
+				
+				// Hide certain columns for feeds
+				let elems = treecols.getElementsByAttribute('hidden-in', '*');
+				for (let i = 0; i < elems.length; i++) {
+					let shouldHide = elems[i].getAttribute('hidden-in').split(' ')
+						.indexOf(collectionTreeRow.type) != -1;
+					let usuallyHidden = elems[i].hasAttribute('usually-hidden');
```

Comment:
So `usually-hidden` means "manually hidden by the user outside of views that force it hidden"?

If so, I think `user-hidden` might be clearer.